other/materials_designer/create_slab.ipynb

```diff
@@ -42,12 +42,12 @@
    "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    print("Available terminations:", [str(t) for t in terminations])
</pre><p>The numbering needs to be fixed, the logic should be:</p><p> </p><p>2. Find possible terminations and select one</p><ul><li>Create CrystalPlanes</li><li>Get all unique slabs and their corresponding terminations</li><li>Visualize all slabs</li><li>Select termination (or use the first one, with index zero by default)</li></ul><p> </p><p>3. Create a slab for the selected termination</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/227/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>